dev/unix/boot-install.sh

```diff
@@ -24,7 +24,6 @@ notion_info() {
 notion_error() {
   command printf '\033[1;31mError\033[0m: ' 1>&2
   notion_eprintf "$1"
-  notion_eprintf ''
```

Comment:
The other printing methods (including the newly added `notion_request()`) still have this extra empty line. Is there a benefit to having them all be consistent with how they output text, either with or without the additional empty line?

Reply:
It was just a result of tweaking the aesthetics until I could get the error message looking good to my eye:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/307871/52614364-5333c180-2e46-11e9-867c-5860f3f70e2f.png">